packages/core/src/link/Link.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  ComponentType,
+  forwardRef,
+  MouseEvent,
+  ReactElement,
+  useCallback,
+} from "react";
+import cx from "classnames";
+import { IconProps, TearOutIcon } from "@jpmorganchase/uitk-icons";
+import { makePrefixer } from "../utils";
+import { Text, TextProps } from "../text";
+
+import "./Link.css";
+
+const withBaseName = makePrefixer("uitkLink");
+
+/**
+ * Links are a fundamental navigation element. When clicked, they take the user to an entirely different page.
+ *
+ * @example
+ * <LinkExample to="#link">Action</LinkExample>
+ */
+export interface InternalLinkProps extends TextProps<"a"> {
+  IconComponent?: never;
+}
+export interface ExternalLinkProps extends TextProps<"a"> {
+  target: "_blank";
+  /**
+   * Override "tearout" icon.
+   */
+  IconComponent?: ComponentType<IconProps>;
+}
+
+export type LinkProps = ExternalLinkProps | InternalLinkProps;
+
+export const Link = forwardRef<HTMLAnchorElement, LinkProps>(function Link(
+  {
+    IconComponent = TearOutIcon,
+    href,
+    className,
+    children,
+    target = "_self",
+    ...rest
+  },
+  ref
+): ReactElement<LinkProps> {
+  const stopPropagation = useCallback(
```

Comment:
Why do we want to stop propagation?

Reply:
I have no idea. We didn't add this? Any thoughts on who might have added it and why?

Reply:
No idea, I would just remove it